CREDITS.md

```diff
@@ -106,21 +131,27 @@ Contributors to the Firecracker repository:
 * Rolf Neugebauer <neugebar@amazon.com>
 * Sam Jackson <sam@clique.app>
 * Samuel Ortiz <sameo@linux.intel.com>
+* sandreim <54316454+sandreim@users.noreply.github.com>
```

Comment:
This looks to be an error.

Reply:
why exactly? I think @sandreim chose to not expose his email on github

Reply:
His address, the correct one, is above

Reply:
will fix it

Reply:
done